lib/repo_tools.pm

```diff
@@ -469,6 +469,11 @@ sub validate_repo_properties {
     }
 
     if ($args->{Name}) {
+        # TODO remove workaround poo#70546
+        if ($actual_repo_data->{Name} =~ /$args->{Alias}\/?/) {
+            record_soft_failure "repo name is not set correctly - bsc#1175374 - found actual repo name $actual_repo_data->{Name}";
+            return;
```

Comment:
I would still validate other fields of the repo, so rather do `if-else` instead of early return. Makes sense?

Reply:
totally and it is done